### DISCUSSION TITLE
How to manual set API key for  service

### DISCUSSION THREAD

**Discussion by shestakow1993** [exadel-inc/CompreFace#970] [Q&A]
We have two fault-tolerant installations and plan to use nginx reverse proxy to access the API, in this case we need services with the same API - key . Is it possible to manually set api key for the service? To have the same API key on two installations

**Comment by pospielov** [comment]
There is no such feature in CompreFace
However, I can recommend two ways:
1. The most proper way here is to take the PostgreSQL database backup and restore it on the second installation. So the API key will be copied with the backup.
2. Create a service and manually change the API key in the PostgreSQL database or with migration.
